_posts/2019/2019-03-13-my-switch-from-js-to-ruby.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: My switch from JavaScript to Ruby
+date: 2019-03-13
+header_image: public/my-switch-from-js-to-ruby.jpg
+header_position: center
+category: tech-stories
+tags: ["javascript", "ruby", "development"]
+authors: ["German"]
+about_authors: ["gsanemeterio"]
+---
+
+Hello, my name is German, I am currently part of ePages as software engineer with ruby as language, although before I worked with Angular.
+When the opportunity of work in ePages came to me it cost me to take a decision but at the end I take the risk, as I like new challenges and to learn new things and I saw it was a great opportunity.
+
+Now in ePages I started with the famous [Rails tutorial of Michael Hartl](https://www.railstutorial.org/){:target="_blank"}.
+During 3 weeks I was reading and trying it at the same time that I was doing the exercises.
+It's amazing how fast things can be done in rails!
+One finished the tutorial, I started to do pair programming with my teammate.
+He explained me the projects and cleared my dubs about ruby and, shortly after, I started with my own tasks, and apply that I learned in the tutorial.
+
+While I was doing the tutorial I was seeing the similar things between rails and another languages with I work before or that I've seen such as Symfony.
+Due to my experience with other languages I could associate concepts of rails with other languages that I know.
+While I was doing the tutorial I was surprise with the easier that rails let to manage the database, thanks to ActiveRecord
+
+As each language there're differences and js and ruby couldn't been the exception, still today I'm thinking in how I do in javascript and then I search ruby's equivalent.
+
+Some of syntax differences with I found are:
+
```

Comment:
You do list items here. Does each of the following examples refer to one of these items? Them better write the item and then have it followed by the example.